griptape/tools/base_aws_client.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING
```

Comment:
this file continues to use the `if TYPE_CHECKING` `import boto3` pattern, but elsewhere we've stopped doing that in favor of importing a `mypy_boto3`. Why not in this file?